### DISCUSSION TITLE
[Release] Facility

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#300] [Map Releases]
I have not made this map, Naievil did, I have merely updated the waypoints, so you can play it on the newest NZP versions.

Edit: I was unaware that this map already had functioning waypoints, so _whoops..._

Link to the original post: https://nzportable.forumotion.com/t2003-release-facility-map-update-v1-02

# Screenshots of this map
![fte-20230521131800-0](https://github.com/nzp-team/nzportable/assets/104042185/9ddd14ec-3d5c-4309-9a7e-eb3e83b65b6b)
![fte-20230521131809-0](https://github.com/nzp-team/nzportable/assets/104042185/e51688a0-fd3b-4924-beed-c31cc28a6fc5)
![fte-20230521131819-0](https://github.com/nzp-team/nzportable/assets/104042185/13d63113-db93-4979-9265-b721cc899e60)
![fte-20230521131825-0](https://github.com/nzp-team/nzportable/assets/104042185/6ab77095-79dd-4e40-bdc1-c9406917adfb)
![fte-20230521131831-0](https://github.com/nzp-team/nzportable/assets/104042185/a58f392d-dfdf-45ae-bece-27dac2427dba)
![fte-20230521131837-0](https://github.com/nzp-team/nzportable/assets/104042185/e00b186d-e1c8-4222-978e-98414fa5dfc4)
![fte-20230521131842-0](https://github.com/nzp-team/nzportable/assets/104042185/9e578de8-cd87-4978-a049-1731001af271)

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/11524365/Facility.zip)

**Comment by dazz88ler** [comment]
i just want to say that this is a really great game

**Reply by MotoLegacy** [reply]
owo

**Comment by Naievil** [comment]
> I have not made this map, Naievil did, I have merely updated the waypoints, so you can play it on the newest NZP versions.
> 
> Edit: I was unaware that this map already had functioning waypoints, so _whoops..._
> 
> Link to the original post: https://nzportable.forumotion.com/t2003-release-facility-map-update-v1-02
> 
> 
> # [DOWNLOAD](https://github.com/nzp-team/nzportable/files/11524365/Facility.zip)

Not to worry. Thanks for wanting to make waypoints for it.

**Reply by Paindeluxe** [reply]
I actually only made them because Ellis from the discord told me in dms that it’s waypoints were broken and I just wanted to be kind and make him some so he can play it also it is a very good map.

**Comment by dazz88ler** [comment]
does anyone know the id for weapons?

**Reply by BCDeshiG** [reply]
See this file: https://github.com/nzp-team/quakec/blob/main/source/shared/defs/custom.qc

**Reply by dazz88ler** [reply]
thx, but i dont know how to work it, cuz every time when i do it it says it failed to execute. how do i fix this?

**Reply by BCDeshiG** [reply]
Not sure what you're trying to do, but the file lists the different weapon IDs, for example ID number 33 corresponds to the pack-a-punched raygun. Usually you'd only need to know the weapon ID when doing the `give` command or setting up wall weapons when making custom maps.
